game/scripts/npc/abilities/pangolier_heartpiercer_oaa.txt

```diff
@@ -0,0 +1,46 @@
+"DOTAAbilities"
+{
+//=================================================================================================================
+// Pangolier: Heartpiercer (OAA rework)
+//=================================================================================================================
+"oaa_heartpiercer"
```

Comment:
Can you name it `pangolier_heartpiercer_oaa`? It's both the name of the file and also a future test case will case.